doctor/views.py

```
@@ -0,0 +1,339 @@
+            serializer.validated_data
+        )
+        
+        if success:
+            return Response(
+                api_ingress.build_success_response(
+                    result,
+                    "پروفایل پزشک با موفقیت ایجاد شد"
+                ),
+                status=status.HTTP_201_CREATED
+            )
+        else:
```

Ruff (0.12.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)